.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
You might want to use this template for `.gitignore` https://github.com/github/gitignore/blob/master/Node.gitignore